README.rst

```diff
@@ -43,6 +43,15 @@ This exporter takes the Draft.js ContentState data as input, and outputs HTML ba
 
     pip install draftjs_exporter
 
+You will also need to install a backing engine. The exporter supports html5lib via BeautifulSoup, and lxml. lxml is the most performant, but it requires ``libxml2`` and `libxslt`` to be available on your system.
+
+.. code:: sh
+
+    # For html5lib,
+    pip install 'html5lib>=0.999,<=1.0b10' 'beautifulsoup4>=4.4.1,<5'
+    # For lxml,
+    pip install lxml
```

Comment:
Changed `pip install draftjs_exporter` back to installing html5lib + BS4, and we now have `pip install drafjts_exporter[lxml]` as an "advanced usage" scenario.